slice-core-api/src/main/java/com/cognifide/slice/api/injector/InjectorConfig.java

```diff
@@ -89,8 +95,19 @@ public String getBundleNameFilter() {
 		return bundleFilter;
 	}
 
+    /**
+     * @deprecated The application path is made multi value, please use ${@link #getApplicationPaths()} from now on.
+     */
+    @Deprecated
 	public String getApplicationPath() {
-		return applicationPath;
+		if(!applicationPaths.isEmpty()) {
```

Comment:
Hmm I don't see it? In the constructor I have a method getApplicationPaths, and if the applicationPaths passed to the constructor is null, i make the field a singleton list of the default injector path (line 67 to 73). I can check for null in this function but it should never happen because of this